components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -332,8 +333,8 @@ void BraveProfileSyncServiceImpl::GetSettingsAndDevices(
 void BraveProfileSyncServiceImpl::GetSyncWords() {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   // Ask sync client
```

Comment:
this comment is now obsolete

Reply:
Minor notice
https://github.com/brave/brave-core/blob/brave-crypto-for-sync/components/brave_sync/brave_sync_service_unittest.cc#L88
```
// OnSyncWordsPrepared       | BraveSyncServiceTest.GetSyncWords
```
this comment is also obsolete.